roles/neutron-common/templates/etc/neutron/neutron.conf

```diff
@@ -109,6 +119,3 @@ rabbit_host = {{ endpoints.rabbit }}
 {% endif -%}
 rabbit_userid = {{ rabbitmq.user }}
 rabbit_password = {{ secrets.rabbit_password }}
-
-[oslo_middleware]
-enable_proxy_headers_parsing = True
```

Comment:
The reason to take out this was, https://github.com/openstack/oslo.middleware/blob/stable/mitaka/oslo_middleware/http_proxy_to_wsgi.py, stable/mitaka doesn't have it. However, I notice that it is added back in master. So should we keep it or not?